src/views/Guide.vue

```diff
@@ -0,0 +1,213 @@
+                </td>
+                <td>
+                  <span>{{ state }}</span>
+                </td>
+                <td style="font-size: 2em;">
+                  <job :status="state" />
+                </td>
+              </tr>
+            </table>
+          </v-card-text>
+        </v-card>
+      </v-flex>
+
+      <v-flex
+        md5
+        xs8
+      >
+        <v-card>
+          <v-card-title primary-title>
+            <p class="display-1 text--primary">Special Task States</p>
+          </v-card-title>
+          <v-card-text>
+            <div id="task-job-special-grid">
+              <div style="font-size: 2em">
+                <task
+                  status="waiting"
+                  :isHeld="true"
+                />
+              </div>
+              <div>
+                <span>Held</span>
+                <br />
+                <span>
+                  When a task is "held" no new job submissions will be made
+                </span>
+              </div>
+            </div>
+          </v-card-text>
+        </v-card>
+      </v-flex>
+    </div>
+
+    <v-divider></v-divider>
+
+    <h2 id="task-and-job-states">
+      Some Other Section
+    </h2>
+
+    <div id="card-grid">
+
+      <v-flex
+        md6
+        xs12
+      >
+        <v-card>
+          <v-card-title>
+            <p class="display-1 text--primary">Foo</p>
+          </v-card-title>
+          <v-card-text>
+            foo
+          </v-card-text>
+        </v-card>
+      </v-flex>
+
+    </div>
+
+  </v-container>
+</template>
+
+<script>
+import Task from '@/components/cylc/Task'
+import Job from '@/components/cylc/Job'
+
+export default {
+  components: {
+    task: Task,
+    job: Job
+  },
+  // TODO: extract task states and descriptions from the GraphQL API
+  //       once this is an enumeration.
+  data: () => ({
+    states: [
+      'waiting',
+      'submitted',
+      'running',
+      'succeeded',
+      'failed',
+      'submit-failed'
+    ]
+  })
+}
+</script>
+
+<style lang="scss">
+  @import '~@/styles/index.scss';
+
+  #card-grid {
+    display: flex;
+    flex-wrap: wrap;
+    margin-bottom: 1em;
+
+    > * {
+      margin: 0.5em;
+    }
+
+  }
+
+  #task-job-state-table {
+    text-align: center;
+    /*border-spacing: 1em 0.5em;*/
+    border-spacing: 0;
+
+    p {
+      font-size: 0.6em;
+      line-height: 1.2em;
+      /*color: $grey;*/
+    }
+
+    tr:nth-child(1) {
+      font-size: 2em;
+    }
+
+    tr > td:nth-child(2) {
+      font-size: 1em;
+    }
+
+    tr > td:nth-child(1), tr > td:nth-child(3) {
+      width: 5em;
+    }
+
+    td {
+      padding: 0.1em 0 0.1em 0;
+    }
+
+    td > * {
+      background-color: white;
+    }
+  }
+
+  #task-job-special-grid {
+    font-size: 1.2em;
+    display: flex;
+    align-items: center;
+
+    div:first-child {
+      padding-right: 0.5em;
+    }
+
+    div > span:nth-child(3) {
+      color: $grey;
+    }
+  }
+</style>
```

Comment:
Maybe we can use Vuetify [typography](https://v15.vuetifyjs.com/en/framework/typography) and [flex grid](https://v15.vuetifyjs.com/en/framework/grid) (will change a bit when we have vuetify 2)?

Removing the styles from here, and using the code from this PR https://github.com/oliver-sanders/cylc-web/pull/1

- Before & after diff desktop:

![image](https://user-images.githubusercontent.com/304786/64387961-aa72db00-d091-11e9-9563-c7237ceaf6b8.png)

![image](https://user-images.githubusercontent.com/304786/64392897-f4b08800-d0a2-11e9-8320-24f614cf452d.png)

- Before & after diff mobile:

![image](https://user-images.githubusercontent.com/304786/64388078-0e959f00-d092-11e9-8581-ada826502bfe.png)

![image](https://user-images.githubusercontent.com/304786/64392893-ec584d00-d0a2-11e9-9f07-505e5a6a9cab.png)

Might still need some adjustments, but that would be easier to be adjusted to Vuetify 2 too (WIP #11).

(Also added some review comments showing what I did basically)

Reply:
> Maybe we can use Vuetify typography and flex grid (will change a bit when we have vuetify 2)?

Any reason to change from using a table? Happy to use Vuetify components if they are higher quality, but in this case I don't think they are.

Reply:
No special reason. I only used these Vuetify components because they translate to `flex`, `flex-wrap`, `nowrap`, and flex columns with `%`s, etc, and I was using them in the Tree view recently. I can try to achieve the same with a table :+1: